core/src/dird/ua_select.h

```diff
@@ -54,6 +54,9 @@ bool SelectClientDbr(UaContext* ua, ClientDbRecord* cr);
 void StartPrompt(UaContext* ua, const char* msg);
 void AddPrompt(UaContext* ua, const char* prompt);
 void AddPrompt(UaContext* ua, std::string&& prompt);
+std::string FormatMulticolumnPrompts(const UaContext* ua,
+                                     const int windowsize,
+                                     const int min_lines_threshold);
```

Comment:
```suggestion
                                     int windowsize,
                                     int min_lines_threshold);
```
we usually don't declare by-value function parameters `const`